src/Scaffolding/VS.Web.CG.Mvc/Templates/MinimalApi/MinimalApi.cshtml

```diff
@@ -103,14 +112,22 @@ public static class @endPointsClassName
 
         group.MapDelete("/{id}", (int id) =>
         {
-            //return Results.Ok(new @modelName { ID = id });
+        @{
+        if(Model.OpenAPI)
+        {
+            @://return TypedResults.Ok(new @modelName { ID = id });
+        }
+        else
+        {
+            @://return Results.Ok(new @modelName { ID = id });
+        }
+        }
         })
     @{
     if(Model.OpenAPI)
     {
-        @:.WithTags(nameof(@modelName))
-        @:.WithName("@deleteModel")
-        @:.Produces<@modelName>(StatusCodes.Status200OK);
+        @:.WithOpenApi()
+        @:.WithName("@deleteModel");

```

Comment:
These should all be reversed so that `WithName` is before `WithOpenApi`

Reply:
yep ty fixed.